### ISSUE TITLE
more explicit errors on stack rollback

### ISSUE THREAD

**Issue by errordeveloper** [weaveworks/eksctl#190]
Right now all one sees on stack rollback is this:

```
$ ./eksctl create cluster
...
2018-09-06T07:51:00+01:00 [ℹ]  creating cluster stack "eksctl-amazing-rainbow-1536216660-cluster"
2018-09-06T07:51:17+01:00 [ℹ]  1 error(s) occurred and cluster hasn't beend created properly, you may wish to check CloudFormation console
2018-09-06T07:51:17+01:00 [ℹ]  to cleanup resources, run 'eksctl delete cluster --region=eu-west-1 --name=amazing-rainbow-1536216660'
2018-09-06T07:51:17+01:00 [✖]  waiting for CloudFormation stack "eksctl-amazing-rainbow-1536216660-cluster" to reach "CREATE_COMPLETE" status: ResourceNotReady: failed waiting for successful resource state
2018-09-06T07:51:17+01:00 [✖]  failed to create cluster "amazing-rainbow-1536216660"
$
```

Stack rollback are often caused by shortage of VPCs:

```

07:51:04 UTC+0100 | CREATE_FAILED | AWS::EC2::VPC | VPC | The maximum number of VPCs has been reached. (Service: AmazonEC2; Status Code: 400; Error Code: VpcLimitExceeded; Request ID: b6615422-7265-4a2c-8098-8914343306df)
-- | -- | -- | -- | --
```

We should be able to detect when it's the cause and print a more friendly error message.

**Comment by tedmiston** [comment]
Unrelated to the rollback itself, but I came here searching for the "cluster hasn't beend created" (sic) error — looks like the typo itself and some others were recently fixed by @kschumy via https://github.com/weaveworks/eksctl/commit/8a65a87d7fc3ea4d206706ed71ba999803f27804.

**Comment by errordeveloper** [comment]
This hides many important errors, such as #200. We must fix this in 0.1.2 asap.
